internal/compiler/passes/ensure_window.rs

```diff
@@ -58,6 +58,7 @@ pub fn ensure_window(
     };
     let new_root = new_root.make_rc();
     win_elem_mut.children.push(new_root.clone());
+    win_elem_mut.node = vec![];
```

Comment:
actually, it would be better to do 
`node: std::mem::take(&mut win_elem_mut.node),`  a few lines earlier rather than cloning and then reseting.   (just like we already do for the children a bit earlier)